content/docs/how-we-work/local-tools.md

```diff
@@ -20,6 +20,31 @@ PHPStorm is perfect for working with core PHP and PHP frameworks such as Drupal,
 
 Visual Studio Code is a free and open-source editor that supports a broad range of programming languages. Visual Studio Code features a rich set of tools that enhance our productivity, such as an integrated terminal, Git integration, and IntelliSense, which provides real-time suggestions and code completion.
 
+#### Extensions/Plugins
+
+Like choosing an IDE that works for you is important, it is also equally important you have the necessary extensions/plugins that works well with your IDE and caters to your development workflow.
+
+As a developer some of the most common tasks done are,
+
+- using GitHub UI to interact with Pull requests (PRs) created by peers like review, merging or closing
+- updating the ticket status and progress in a tool like JIRA (there are other tools as well however we use JIRA at Axelerant).
+
+Using extensions to do the above right from the IDE allows for better collaboration, less turn around times for PR reviews and more accurate time logging and ticket updates.
+
+Below are some recommendations for the above tasks.
+
+##### GitHub extensions
+
+- VS Code - [Manual](https://code.visualstudio.com/docs/sourcecontrol/github) with [demos](https://vscode.github.com/)
+- PHP Storm - [Manual](https://www.jetbrains.com/help/phpstorm/github.html)
+
+##### JIRA extensions
+
+- VS Code - [This extension](https://marketplace.visualstudio.com/items?itemName=gioboa.jira-plugin&ssr=false#overview) has better reviews than [Atlassian extension](https://marketplace.visualstudio.com/items?itemName=Atlassian.atlascode&ssr=false#overview)
+- PHP Storm - [This extension](https://plugins.jetbrains.com/plugin/11169-jira-integration) though there are [mixed reviews](https://plugins.jetbrains.com/plugin/11169-jira-integration/reviews) and some [issues](https://stackoverflow.com/questions/41593590/phpstorm-and-jira-integration), [this video](https://www.youtube.com/watch?v=uQuqIMtLrB0) might help
```

Comment:
@Kalaiselvan88, can you mention a shortened version of the extension name rather than just saying "This extension"? It will improve readability, I think.

Also, I feel that the description is potentially confusing. It makes it seem like the mixed reviews are a deal breaker and they should look elsewhere. Can you shorten it to only the necessary information they need here? If they care about reviews, they will find it on the extension page.

Reply:
Updated in 3c4b84d3faf44b8fc75227818bb31998aca02552